runtime/syntax/ppd.vim

```diff
@@ -15,7 +15,7 @@ syn match	ppdDefine	"\*[a-zA-Z0-9\-_]\+:"
 syn match	ppdUI		"\*[a-zA-Z]*\(Open\|Close\)UI"
 syn match	ppdUIGroup	"\*[a-zA-Z]*\(Open\|Close\)Group"
 syn match	ppdGUIText	"/.*:"
-syn match	ppdContraints	"^*UIConstraints:"
+syn match	ppdConstraints	"^*UIConstraints:"
```

Comment:
This may be a typo in PostScript.

Reply:
No! This is vim ID.
http://www.setec.org/~izaac/postscriptbooks/partners.adobe.com/asn/developer/pdfs/tn/5003.PPD_Spec_v4.3.pdf